### DISCUSSION TITLE
Include newest version of YTLite (5.0b)

### DISCUSSION THREAD

**Discussion by schweppes-0x** [YTLitePlus/YTLitePlus#381] [Ideas]
I received a .deb file for the newest YTLite version, could be nice to include this in the next release

**Comment by Balackburn** [comment]
You can already use the 5.0b branch when you build instead of main

**Reply by schweppes-0x** [reply]
what is the reason the main branch is not using the latest version ? :)

**Reply by Balackburn** [reply]
> what is the reason the main branch is not using the latest version ? :)

Because latest version is a paid beta
